tests/apichecks/crds_test.go

```diff
@@ -526,6 +526,14 @@ func TestCRDFieldPresenceInUnstructured(t *testing.T) {
 					}
 				}
 
+				// Exclude output-only spec fields.
+				switch crd.Name {
```

Comment:
Same comment I made in another PR. It is probably easier to maintain if we write all output only spec fields to a golden file.

Reply:
Done!